### PR TITLE
secret token retrieval should be the same as the docs

### PR DESCRIPTION
**Pull request by tsuz**
Fixes: https://github.com/mapbox/mapbox-android-demo/issues/1393

It should follow the same procedure as the instruction step 5.

https://docs.mapbox.com/android/maps/guides/install/#add-the-dependency

### CODE REVIEW

**Review by moritzzzzz — APPROVED** [review]

**Review by Chaoba — APPROVED** [review]
Thanks for your contribution.

**Comment by dudeuter** [comment]
Just ran into this myself

CI needs `MAPBOX_DOWNLOADS_TOKEN` for the test to pass.

**Comment by tsuz** [comment]
@dudeuter Thanks. I thought I merged this already..

@Chaoba Can you help us update the env variable of `MAPBOX_DOWNLOADS_TOKEN`?

File: build.gradle [hunk 1]
```diff
@@ -39,7 +39,7 @@ allprojects {
             }
             credentials {
                 username = "mapbox"
-                password = System.getenv("SDK_REGISTRY_TOKEN") ?: project.property("SDK_REGISTRY_TOKEN") as String
+                password = project.properties['MAPBOX_DOWNLOADS_TOKEN'] ?: ""
```
Comment (Chaoba) [inline build.gradle:42]: We must keep `System.getenv` to get the token in CI. Can you add it?

**Comment by Chaoba** [comment]
Close this as fixed in https://github.com/mapbox/mapbox-android-demo/pull/1401

**Pull request closed without merging**